src/test/scala/com/redislabs/provider/redis/util/TestUtils.scala

```diff
@@ -13,4 +16,24 @@ object TestUtils {
     UUID.randomUUID().toString.replace("-", "")
   }
 
+  /**
+    * A wrapper of Assertions.intercept() that suppresses spark errors in the logs.
+    * It makes easier to analyse unit tests output.
+    */
+  def interceptSparkErr[T <: AnyRef](f: => Any)(implicit manifest: Manifest[T]): T = {
+    // turn off spark logger
+    val logger = org.apache.log4j.Logger.getLogger("org")
```

Comment:
@fe2s why "org"?

Reply:
It's mainly "org.apache.spark" package and might be other classes from "org." package